core/src/main/java/org/stellar/anchor/sep31/RefundsBuilder.java

```diff
@@ -27,4 +30,34 @@ public RefundsBuilder payments(List<RefundPayment> payments) {
   public Refunds build() {
     return refunds;
   }
+
+  /**
+   * loadPlatformApiRefunds will load the values from the PlatformApi Refund object into the SEP-31
+   * Refunds object.
+   *
+   * @param platformApiRefunds is the platformApi's Refund object.
+   * @return a SEP-31 Refunds object.
+   */
+  public Refunds loadPlatformApiRefunds(org.stellar.anchor.api.shared.Refund platformApiRefunds) {
+    if (platformApiRefunds == null) {
+      return null;
+    }
+
+    ArrayList<RefundPayment> payments = null;
+    for (org.stellar.anchor.api.shared.RefundPayment platformApiRefundPayment :
+        platformApiRefunds.getPayments()) {
+      if (payments == null) {
```

Comment:
This check seems necessary, maybe you can remove it?

Reply:
We do need to initialize the `payments` list before calling payments.add though:

https://github.com/stellar/java-stellar-anchor-sdk/blob/0450497e53516b3148118eef5e08be43ce218f6a/core/src/main/java/org/stellar/anchor/sep31/RefundsBuilder.java#L46-L56

I updated the code in https://github.com/stellar/java-stellar-anchor-sdk/pull/448/commits/a3e00a22ab93f3e56c5096d568081effd92b23f0 to initialize it once outside the loop instead of checking for it on every loop lap.